### PR TITLE
updated support questions

### PR DESCRIPTION
**Pull request by jerae-duffin**
Updated support questions into Docs

### CODE REVIEW

File: docker-hub/onboarding-faqs.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ For more information, see [Docker ID](../docker-id/index.md). If your admin enfo
 
 ### What if my Docker ID is taken?
 
-If you have a trademark for your namespace, [Docker Support](https://hub.docker.com/support/contact/){: target="_blank" rel="noopener"
+All Docker IDs are first-come, first-served except for companies that have a US Trademark on a username. If you have a trademark for your namespace [Docker Support](https://hub.docker.com/support/contact/){: target="_blank" rel="noopener"
```
Comment (usha-mandya) [inline docker-hub/onboarding-faqs.md:16]:
```suggestion
All Docker IDs are first-come, first-served except for companies that have a US Trademark on a username. If you have a trademark for your namespace, [Docker Support](https://hub.docker.com/support/contact/){: target="_blank" rel="noopener"
```

File: docker-hub/repos.md [hunk 2]
```diff
@@ -25,8 +25,10 @@ When creating a new repository:
 * You can choose to put it in your Docker ID namespace, or in any
   [organization](orgs.md) where you are an [_owner_](orgs.md#the-owners-team).
 * The repository name needs to be unique in that namespace, can be two
-  to 255 characters, and can only contain lowercase letters, numbers, hyphens (`-`),
-  and underscores (`_`).
+  to 255 characters, and can only contain lowercase letters, numbers, hyphens (`-`), and underscores (`_`).
```
Comment (usha-mandya) [inline docker-hub/repos.md:28]: Not sure if underscores are allowed. @siarheikarneichyk Could you confirm?

File: docker-hub/repos.md [hunk 3]
```diff
@@ -25,8 +25,10 @@ When creating a new repository:
...
+
+  **Note:** you cannot rename a Docker Hub repository once it has been created.
```
Comment (usha-mandya) [inline docker-hub/repos.md:30]: Update to use the standard note format

File: docker-hub/repos.md [hunk 4]
```diff
@@ -36,6 +38,11 @@ When creating a new repository:
 After you hit the **Create** button, you can start using `docker push` to push
 images to this repository.
 
+
+## Deleting a repository
+
+ Sign into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} with your user account to delete a repository. Navigate to the repository **Settings** and click **Delete Repository** to delete the repository and all images stored within it.
```
Comment (usha-mandya) [inline docker-hub/repos.md:44]: I think the order is:
1. Sign into Docker Hub.
2. Click Repositories.
3. Select a repository from the list and click **Settings** and then **Delete Repository**. Deleting a repository deletes all the images it contains and its build settings. This action cannot be undone.
4. Enter the name of the repository to confirm the deletion.
5. Click **Delete**.

File: docker-id/index.md [hunk 5]
```diff
@@ -17,11 +17,11 @@ Your Docker ID becomes your user namespace for hosted Docker services, and becom
 
 2. Enter a username that will become your Docker ID.
 
-    Your Docker ID must be between 4 and 30 characters long, and can only contain numbers and lowercase letters.
+    Your Docker ID must be between 4 and 30 characters long, and can only contain numbers and lowercase letters. Once you create your Docker ID you can not reuse it in the future if you deactivate this account.
```
Comment (usha-mandya) [inline docker-id/index.md:20]:
```suggestion
    Your Docker ID must be between 4 and 30 characters long, and can only contain numbers and lowercase letters. Once you create your Docker ID you cannot reuse it in the future if you deactivate this account.
```

File: single-sign-on/index.md [hunk 6]
```diff
@@ -188,3 +188,9 @@ To remove a member from an organization:
 
 ## FAQs
 To learn more see our [FAQs](faqs.md).
+
+## Deleting SSO
+
+When SSO is disabled, you can delete the connection to remove the configuration settings and the added domains. Once you delete this connection, it cannot be undone. Users must authenticate with their Docker ID and password or create a password reset if they do not have one.
```
Comment (usha-mandya) [inline single-sign-on/index.md:194]:
```suggestion
When you disable SSO, you can delete the connection to remove the configuration settings and the added domains. Once you delete this connection, it cannot be undone. Users must authenticate with their Docker ID and password or create a password reset if they do not have one.
```

File: subscription/upgrade.md [hunk 7]
```diff
@@ -60,6 +60,6 @@ To update the billing information for your organization or personal account:
 
     > **Note**
     >
-    > The details displayed in the **Contact Information** section is used to generate your invoice. All billing-related information is sent to the email address listed in this section.
+    > The details displayed in the **Contact Information** section is used to generate your invoice. All billing-related information is sent to the email address listed in this section. Invoices are sent to one email address. Please contact your admistrator to create a group within your organization that includes all of the email addresses.
```
Comment (usha-mandya) [inline subscription/upgrade.md:63]: typo: administrator

File: subscription/upgrade.md [hunk 8]
```diff
@@ -60,6 +60,6 @@ To update the billing information for your organization or personal account:
...
 
 4. Click **Update** to save your changes.
```
Comment (usha-mandya) [inline subscription/upgrade.md:65]: Add a blank line at the end of the markdown file.

File: docker-hub/onboarding-faqs.md (hunk 1 above)
Comment (jerae-duffin) [reply docker-hub/onboarding-faqs.md:16]: Updated, thank you!

File: docker-hub/repos.md (hunk 3 above)
Comment (jerae-duffin) [reply docker-hub/repos.md:30]: Updated

File: docker-id/index.md (hunk 5 above)
Comment (jerae-duffin) [reply docker-id/index.md:20]: Updated

File: single-sign-on/index.md (hunk 6 above)
Comment (jerae-duffin) [reply single-sign-on/index.md:194]: Updated

File: subscription/upgrade.md (hunk 7 above)
Comment (jerae-duffin) [reply subscription/upgrade.md:63]: Good catch, thank you!

File: subscription/upgrade.md (hunk 8 above)
Comment (jerae-duffin) [reply subscription/upgrade.md:65]: Updated

File: docker-hub/builds/link-source.md [hunk 9]
```diff
@@ -32,6 +32,10 @@ organizations.
 
     ![Linking source providers](images/linked-accounts.png)
 
+    > **Note**:  If you want to unlink your current GitHub account and relink to a new GitHub account, make sure to completely logout of [GitHub] (https://github.com/){: target="_blank" rel="noopener"
```
[the remark this answers is not in the document]
Comment (jerae-duffin) [reply docker-hub/builds/link-source.md:35]: Updated

File: docker-hub/repos.md (hunk 4 above)
Comment (jerae-duffin) [reply docker-hub/repos.md:44]: Updated

File: docker-hub/repos.md (hunk 2 above)
Comment (siarheikarneichyk) [reply docker-hub/repos.md:28]: Underscores are allowed. But repo name can't end with an underscore or a hyphen. So `test_repo` or `test-repo` is fine. But `test_repo_` ore `test-repo-` won't work. Idk if we want to mention that.

Comment (jerae-duffin) [reply docker-hub/repos.md:28]: Thanks! @siarheikarneichyk Updated

Comment (usha-mandya) [reply docker-hub/repos.md:28]: Thank you @siarheikarneichyk

**Review by usha-mandya — APPROVED** [review]
LGTM

**Comment by usha-mandya** [comment]
@siarheikarneichyk Now this PR is merged, can we remove the Hub support page and add a redirect to the Docs?

**Comment by siarheikarneichyk** [comment]
@usha-mandya I think Jerae is still working on moving all the popular questions from hub support to docs. After that, we can update links.

**Comment by usha-mandya** [comment]
Sure, thank you @siarheikarneichyk

**Pull request merged**